docpages/INDEX.md

```diff
@@ -36,10 +36,10 @@ You can find further releases in other architectures and formats or the source c
 * Efficient caching system for guilds, channels, guild members, roles, users
 * Sharding and clustering (Many shards, one process: specify the number of shards, or let the library decide)
 * Highly optimised ETF (Erlang Term Format) support for very fast websocket throughput (*no other C++ Discord library has this!*)
-* [Slash Commands/Interactions support](https://dpp.dev/slashcommands.html)
-* [Voice support](https://dpp.dev/soundboard.html) (sending **and** receiving audio)
+* \ref slashcommands "Slash Commands/Interactions support"
```

Comment:
if we are doing this please make sure the names of the links match what they used to be . usually when doing \ref, the title of the link is always the title of the page but this breaks the sentences where we put a link in a sentence.